File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { UserAgentLookupSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await UserAgentLookupSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- UserAgentLookup SDK exists test

local sdk = require("user-agent-lookup_sdk")

describe("UserAgentLookupSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
